benchmarks/test_logprobs_stream_native.py

```
@@ -0,0 +1,474 @@
+                    "total_tokens": prompt_tokens + total_tokens,
+                    "prompt_tokens_details": {"cached_tokens": cached_tokens},
+                }
+
+            expected.append(chunk)
+            cursor[output_idx] = len(token_ids)
+    return expected
+
+
+@dataclass
+class _NativeReplay:
+    handlers: dict[int, AggregatedHandler]
```

Ruff (0.16.3)

[error] Undefined name `AggregatedHandler`

(F821)